core/templates/dev/head/pages/error/error.html

```diff
@@ -67,10 +67,8 @@ <h2 class="oppia-error-h2" translate="I18N_ERROR_MESSAGE_500"></h2>
     .oppia-footer {
       position: absolute;
     }
-    @media (max-width: 768px) {
-      .oppia-footer {
-        position: relative;
-      }
+    .oppia-wide-panel-content {
+      margin-bottom: 15em
```

Comment:
All mobile resolutions. 
Tip: Refresh the page once you are in the mobile view. Just making it responsive won't render it correctly.  
Let me know if you reproduced at your end. 

P.S. I tested on OnePlus 3, Chrome browser.